packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -1,12 +1,22 @@
 const { Command } = require('@oclif/command');
 
-const { startSpinner, endSpinner } = require('../utils/display');
+const { startSpinner, endSpinner, formatStyles } = require('../utils/display');
 
 const inquirer = require('inquirer');
 
+const DATA_FORMATS = ['json', 'raw'];
+
 class ZapierBaseCommand extends Command {
   run() {
     this._parseFlags();
+    if (this.flags.debug) {
+      this.debug.enabled = true; // enables this.debug on the command
+      require('debug').enable('zapier:*'); // enables all further spawned functions, like API
```

Comment:
When I run `zapier versions --debug`, I only see `zapier:versions` logs. Should it also include `zapier:api` logs?

![](https://zappy.zapier.com/FCB24EFE-01FD-4FB3-A2AC-7689D56576E2.png)

Reply:
you should, and that's what I see: ![](https://zappy.zapier.com/35C77E0B-54BF-4D1C-977F-6727AFEC5530.png)

that was running `../../../platform/packages/cli/zapier.js versions --debug`

Reply:
works as expected when running via `~/projects/zapier/platform/node_modules/.bin/zapier versions --debug` as well 🤔 

If you've got a sec, can you add the following line to the top of the `callAPI` method in `cli/src/utils/api.js`? Should be line 45 of that file.

```js
console.log(require('debug').names, debug.enabled);
```

When you run `versions` without `--debug`, it should show `[] false`. When it's true, it should be `[ /^zapier:.*?$/ ] true`

Reply:
NVM, running `yarn` fixes the issue. Looks like I forgot to run `yarn` in `zapier-platform` so `.bin/zapier` wasn't up-to-date.